be/src/olap/memtable.cpp

```diff
@@ -161,11 +179,12 @@ void MemTable::_insert_one_row_from_block(RowInBlock* row_in_block) {
     if (is_exist) {
         _aggregate_two_row_in_block(row_in_block, _vec_hint.curr->key);
     } else {
-        row_in_block->init_agg_places(_agg_functions, _schema->num_key_columns());
+        row_in_block->init_agg_places(
+                (char*)_table_mem_pool->allocate(_total_size_of_aggregate_states),
```

Comment:
How much is _total_size_of_aggregate_states usually, thks

Reply:
I think `_total_size_of_aggregate_states` will not be large, depending on the number and type of value columns. Each value column needs a buffer to store the agg result, which is independent of the number of lines.

Usually the number of value column will not be very large, may be less than 10. And the size and alignment length of basic data types are not large.

However, the cumulative value of memory allocated by mem pool will be relatively large.

Reply:
This is suitable for MemPool~